### PR TITLE
Updated package (to 1.0.17) and using new conan syntax

### PR DESCRIPTION
**Pull request by macat**
- This is now working with conan 1.8
- Updated openvr version to `1.0.17`

### CODE REVIEW

File: conanfile.py [hunk 1]
```diff
@@ -18,39 +18,34 @@ class OpenvrConan(ConanFile):
...
+                              '''project(OpenVRSDK)
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()''')
 
     def build(self):
-        cmake = self._configure_cmake()
+        cmake = CMake(self)
+        cmake.configure(source_folder="openvr")
         cmake.build()
+        cmake.install()
```
Comment (intelligide) [inline conanfile.py]: See <https://bincrafters.readthedocs.io/en/latest/contributing_to_packages/package_guidelines_required.html#build-method>

**Comment by intelligide** [comment]
Thanks for your contribution ! :) 

We are integrating your edits into the main repository located at <https://gitlab.com/ArsenStudio/ArsenEngine/dependencies/conan-openvr>. (This github repo is a read only mirror and we cannot accept pull requests here)

**Comment by macat** [comment]
Sounds good! Thank you!

**Comment by intelligide** [comment]
`1.0.17` is now available in bintray <https://bintray.com/arsen-studio/arsen-deps/openvr%3Aarsen-studio/1.0.17%3Astable>

**Pull request closed without merging**